client/extensions/woocommerce/app/settings/shipping/shipping-zone/shipping-zone-header.js

```diff
@@ -15,11 +15,14 @@ import Button from 'components/button';
 import { getLink } from 'woocommerce/lib/nav-utils';
 import { getCurrentlyEditingShippingZone } from 'woocommerce/state/ui/shipping/zones/selectors';
 import { getSelectedSite } from 'state/ui/selectors';
+import { getActionList } from 'woocommerce/state/action-list/selectors';
+import { areCurrentlyEditingShippingZoneLocationsValid } from 'woocommerce/state/ui/shipping/zones/locations/selectors';
 
-const ShippingZoneHeader = ( { zone, site, onSave, translate } ) => {
+const ShippingZoneHeader = ( { zone, site, onSave, onDelete, translate, isBusy, canSave, isSaving } ) => {
```

Comment:
Nope, I renamed it to `isSaving` and I messed that up in rebases/merges, good catch!